extensions/EXT_color_buffer_half_float/extension.xml

```diff
@@ -37,14 +37,16 @@
         Renderbuffers can be created in these formats. These and textures
         created with <code>type = HALF_FLOAT_OES</code>, which will have one
         of these internal formats, can be attached to framebuffer object color
-        attachments for rendering.</p>
+        attachments for rendering. <code>RGBA16F</code> textures are guaranteed
+	to be color-renderable. <code>RGB16F</code> textures are color-renderable
```

Comment:
I would say:
RGB16F and RGBA16F effective internal formats become color-renderable, but only RGBA16F is guaranteed to return FRAMEBUFFER_COMPLETE from checkFramebufferStatus().